### PR TITLE
Pipelines 5988: Update override for git resolver

### PR DESCRIPTION
**Pull request by nebelsprachen**
Version(s):

* Cherrypick to `pipelines-docs-1.14` `pipelines-docs-1.15` `pipelines-docs-1.16`

Issue:

* https://issues.redhat.com/browse/RHDEVDOCS-5988

Link to docs preview: https://78382--ocpdocs-pr.netlify.app/openshift-pipelines/latest/create/remote-pipelines-tasks-resolvers.html 

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by ppitonak — CHANGES_REQUESTED** [review]

File: modules/op-resolver-git-scm.adoc
Comment (ppitonak) [reply modules/op-resolver-git-scm.adoc:71]: we should use v1 everywher

File: modules/op-resolver-git-anon.adoc
Comment (ppitonak) [inline modules/op-resolver-git-anon.adoc:76]:
```suggestion
      value: "testPipelineRun"
    - name: sample-pipeline-parameter
      value: test
```

File: modules/op-resolver-git-anon.adoc
Comment (ppitonak) [reply modules/op-resolver-git-anon.adoc:73]: see the suggestion below

File: modules/op-resolver-git-anon.adoc
Comment (ppitonak) [inline modules/op-resolver-git-anon.adoc:73]:
```suggestion
```

File: modules/op-resolver-git-anon.adoc
Comment (ppitonak) [inline modules/op-resolver-git-anon.adoc:103]:
```suggestion
      params:        
      - name: sample-task-parameter
        value: test
```

File: modules/op-resolver-git-override-scm.adoc
Comment (ppitonak) [inline modules/op-resolver-git-override-scm.adoc:39]:
```suggestion
    params:
    - name: org
      value: tektoncd
    - name: repo
      value: catalog
    - name: revision
      value: main
    - name: pathInRepo
      value: task/git-clone/0.6/git-clone.yaml
    - name: token
      value: my-secret-token
    - name: tokenKey
      value: token
    - name: scmType
      value: github
    - name: serverURL
      value: https://ghe.mycompany.com
```

**Review by ppitonak — CHANGES_REQUESTED** [review]

**Review by vdemeester — APPROVED** [review]

File: modules/op-resolver-git-override-scm.adoc
Comment (adellape) [inline modules/op-resolver-git-override-scm.adoc:74]: This needs some passthrough to make it non-clickable (like the others in earlier tables):
```suggestion
| `+https://ghe.mycompany.com+`
```

File: modules/op-resolver-git-override-scm.adoc
Comment (adellape) [inline modules/op-resolver-git-override-scm.adoc:46]: Overall, this table should be marked-up/formatted (for file names, field/value names, etc.) to better match the earlier tables, too. For example in this first row (though, apply uniformly throughout the table):
```suggestion
.Supported parameters to override the Git resolver
|===
| Parameter | Description | Example value

| `org`
| The organization for the repository.
| `tektoncd`
```

**Comment by adellape** [comment]
Also reminder to squash down to 1 commit before requesting merge. Thank you!

**Review by ppitonak — APPROVED** [review]

File: modules/op-resolver-git-override-scm.adoc
Comment (adellape) [inline modules/op-resolver-git-override-scm.adoc:73]: I meant for the whole table to get the marked-up treatment (where appropriate), but not going to block merge on this. But consider for a follow-up.

**Review by adellape — APPROVED** [review]

**Pull request merged**
